widgets/WidgetTreeSelector.php

```diff
@@ -464,6 +464,13 @@ protected function renderItemTree($id, $intMargin, $blnNoRecursion=false)
         return $return;
     }
 
+    protected function isLeaf($objItem)
+    {
+        $objChildren = $this->Database->prepare("SELECT id FROM $this->foreignTable WHERE pid=?")->execute($objItem->id);
+
+        return $objChildren->numRows <= 0;
```

Comment:
Can the `numRows` be ever negative? Please change to: `return (int) $objChildren->numRows === 0;`